kv/txn_correctness_test.go

```diff
@@ -178,28 +183,34 @@ func scanCmd(c *cmd, txn *client.Txn, t *testing.T) *roachpb.Error {
 	return nil
 }
 
-// incCmd adds one to the value of c.key in the env and writes
-// it to the db. If c.key isn't in the db, writes 1.
+// incCmd adds one to the value of c.key in the env (as determined by
+// a previous read, or else assumed to be zero) and writes it to the
+// db. If c.key isn't in the db, writes 1.
 func incCmd(c *cmd, txn *client.Txn, t *testing.T) *roachpb.Error {
-	r, pErr := txn.Inc(c.getKey(), 1)
```

Comment:
Using built-in increment instead of reading from the correctness test and then writing was effectively masking problems with transactional correctness which had crept in.